tests/unit/routes/rentals/show-test.js

```diff
@@ -0,0 +1,24 @@
+import { moduleFor, test } from 'ember-qunit';
```

Comment:
We should add a route test that verifies that the model is set to the selected rental.  The slightly funky thing is that in practice we would probably be using something like pretender or mirage to stub these calls, but we are using it anyways for the tutorial.